docs/book/common_code/machine_controller.md

```diff
@@ -10,16 +10,20 @@ the host with a new one matching the updated spec. If a `Machine` object is
 deleted, the corresponding `Node` should have its external resources released by
 the provider-specific controller, and should be deleted as well.
 
+Machines can be associated with a Cluster using a custom label
```

Comment:
do we want to explicitly call out that users shouldn't update this label?

Reply:
Good point, going to update the doc.